docs/src/_data/sites/en.yml

```diff
@@ -26,9 +26,8 @@ google_analytics:
 # Ads
 #------------------------------------------------------------------------------
 
-carbon_ads:
-  serve: CESDV2QM
-  placement: eslintorg
+carbon_ads: false
```

Comment:
We should keep `serve:""` and `placement:""` here to make it easier to switch back (and remember what was required)

Reply:
Changed 👍🏻